src/components/routerLink.browser.spec.ts

```diff
@@ -70,7 +70,9 @@ test.each([
 
   app.find('a').trigger('click')
 
-  expect(spy).toHaveBeenLastCalledWith(routeB.path, { replace })
+  const [, arg2] = spy.mock.lastCall ?? []
+
+  expect(arg2).toMatchObject({ replace, query: undefined })
```

Comment:
This was annoying. Because push is being called with a route method response (which includes functions) `expect(spy).toBeLastCalledWith(router.routes.routeB(), { replace, query: undefined })` failed with the message "Values have no visual difference". They were the same but expect couldn't verify the functions were the same or something. 

Getting the args from last call is pretty messy looking. But it means
- I can check just the options arg
- I can use `toMatchObject` which works best when comparing objects and which there is no `toHaveBeenCalledWith` equivalent.